src/celeritas/field/RZMapFieldInput.hh

```diff
@@ -7,10 +7,12 @@
 //---------------------------------------------------------------------------//
 #pragma once
 
+#include <fstream>
```

Comment:
```suggestion
```

Nothing in this file depends on `fstream`.

Reply:
Removed.